modules/ols-about-lightspeed-and-role-based-access-control.adoc

```diff
@@ -7,6 +7,8 @@
 
 Role-Based Access Control (RBAC) is a system security approach to restricting system access to authorized users who have defined roles and permissions. 
 
```

Comment:
@kyoto it might be helpful if I could list the permission that is required in order for a user to be able to submit a qusetion to Lightspeed.

Reply:
I think we already mention the required role (`lightspeed-operator-query-access`) in the "Granting access to an individual user" section below this section, which is probably sufficient?

Reply:
Thanks for the reminder